code/game/objects/items/melee/batons/classic.dm

```diff
@@ -0,0 +1,133 @@
+/obj/item/melee/classic_baton
+	name = "police baton"
+	desc = "A wooden truncheon for beating criminal scum."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "baton"
+	item_state = "classic_baton"
+	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
+	slot_flags = ITEM_SLOT_BELT
+	force = 12 //9 hit crit
+	w_class = WEIGHT_CLASS_NORMAL
+	var/cooldown = 13
+	var/last_hit = 0
+	var/stun_stam_cost_coeff = 1.25
+	var/hardstun_ds = 1
+	var/softstun_ds = 0
+	var/stam_dmg = 30
+
+/obj/item/melee/classic_baton/attack(mob/living/target, mob/living/user)
+	if(!baton_stun(target, user))
+		return ..()
+
+// return true to disrupt normal melee attack chain
+/obj/item/melee/classic_baton/proc/baton_stun(mob/living/target, mob/living/user)
+	if(user.getStaminaLoss() >= STAMINA_SOFTCRIT)//CIT CHANGE - makes batons unusuable in stamina softcrit
+		to_chat(user, "<span class='warning'>You're too exhausted for that.</span>")//CIT CHANGE - ditto
```

Comment:
`obj/item/attack `already says this line. And maybe remove the CIT CHANGE comments.

Reply:
attack isn't called, batons all have attack snowflaked. this needs to be here otherwise this needs to use pre_attack